pages/balance/2024/3810.html

```diff
@@ -386,7 +394,7 @@ <h2>ACU</h2>
                     <div id="bubble" class="Card">
                         <h3><img src="/assets/images/Enhancements/uef/bubbleshield.png">UEF Bubble-Shield</h3>
     
-                        <p>Reduce the costs of the UEF Bubble Shield Upgrade.</p>
+                        <p>Reduce the cost of the UEF Bubble Shield Upgrade.</p>
```

Comment:
It's probably debatable if 'cost' or 'costs' is more correct here, since there are 3 cost values, but that's the cost of it, either way it should be 'reduced' not 'reduce', since it's past tense.

Reply:
Actually, are these past tense? I just noticed a few of these are talking as if the changes are still to come/instructions.

Reply:
There does not seem to be a system regarding which tenses should be used. I prefer 'cost' for the reason you mentioned but I can change if you disagree.

Reply:
I think that changelogs should be past tense. Otherwise it reads like a list of instructions that someone is to follow and not a list of things that were done. It's confusing enough that it made me question if this was a changelog or a todo list after I saw that it was more than just a missing D.
I don't have strong feelings about cost vs costs in this case.

Reply:
I updated the description of the change; it wasn't mine, so I decided not to interpret too much into the intention behind it.

Unfortunately, the changelog snippet for this balance change did not provide a real description, only the stat changes themselves, which makes it pretty confusing. In general, we should make the intention behind our changes as clear as possible so that players not as involved as we are, can understand them. It also helps us reconstruct the thought process behind older changes in the future.